### PR TITLE
Modify linux wpk generation script to support symbols

### PR DESCRIPTION
**Pull request by HanesSciarrone**
|Related issue|
|---|
|#1309|

## Description

This PR aims to create a debug symbols tar.gz file when a new wpk is created, this file contains the debug symbols of the binary and executable files and will be useful when we want to make a debug session for a specific version.

- Build the package in any supported platform
  - [x] Linux
  - [ ] Windows
  - [ ] macOS
  - [ ] Solaris
  - [ ] AIX
  - [ ] HP-UX
- [x] Package installation
- [ ] Package upgrade
- [ ] Package downgrade
- [ ] Package remove
- [ ] Package install/remove/install
- [ ] Change added to CHANGELOG.md

- Tests for Linux RPM
  - [x] Build the package for x86_64
  - [ ] Build the package for i386
  - [ ] Build the package for armhf
  - [ ] Build the package for aarch64
  - [ ] `%files` section is correctly updated if necessary
- Tests for Linux deb
  - [ ] Build the package for x86_64
  - [ ] Build the package for i386
  - [ ] Build the package for armhf
  - [ ] Build the package for aarch64
  - [ ] Package install/remove/install
  - [ ] Package install/purge/install
  - [ ] Check file permissions after installing the package
- Tests for macOS
  - [ ] Test the package from macOS Sierra to Mojave
- Tests for Solaris
  - [ ] Test the package on Solaris 10
  - [ ] Test the package on Solaris 11
  - [ ] Check file permissions on Solaris 11 template
- Tests for IBM AIX
  - [ ] `%files` section is correctly updated if necessary
  - [ ] Check the changes from IBM AIX 5 to 7

### CODE REVIEW

**Review by RamiroRD — APPROVED** [review]

File: wpk/run.sh [hunk 1]
```diff
@@ -175,7 +175,9 @@ main() {
 
         # Compile agent
         make -C src -j ${JOBS} TARGET=${BUILD_TARGET} || exit 1
-        # Clean unuseful files
```
Comment (jnasselle) [inline wpk/run.sh:178]: This comment should be untouched and located before clean step

File: wpk/run.sh [hunk 2]
```diff
@@ -175,7 +175,9 @@ main() {
...
+        rm -rf ./src/symbols/.gitignore
+        tar -C src/symbols -czf ${OUTDIR}/wazuh-${BUILD_TARGET}-linux-x86_64-debug-info-${WAZUH_VERSION}-${REVISION}.tar.gz .
```
Comment (jnasselle) [inline wpk/run.sh:179]: Set owner and group during tar compression

Comment (jnasselle) [inline wpk/run.sh:179]: Current output is `wazuh-agent-linux-x86_64-debug-info-v4.4.0-1.tar.gz`. Version should not contain `v` prefix

**Review by jnasselle — APPROVED** [review]

File: wpk/run.sh (hunk 2 above)
Comment (jnasselle) [reply wpk/run.sh]: Nevermind.`v` was ok according to wpk name convention

**Review by RamiroRD — APPROVED** [review]

**Review by Dwordcito — APPROVED** [review]

**Pull request merged**